boards/BMS/Inc/App/states/App_ChargeState.h

```diff
@@ -1,9 +1,11 @@
 #pragma once
 
 #include "App_SharedStateMachine.h"
-
+#include "App_Accumulator.h"
+// 0.05C is standard for a boundary to consider full charge
+#define CURRENT_AT_MAX_CHARGE (0.05f * C_RATE_TO_AMPS);
```

Comment:
This isn't used in any other file, so you should define it in the .c file instead to limit scope